project/Build.scala

```diff
@@ -12,7 +12,7 @@ import com.typesafe.sbt.sdlc.Plugin._
 
 object SlickBuild extends Build {
 
-  val slickVersion = "3.2.0-SNAPSHOT"
+  val slickVersion = "3.2.0-M1-lifeonair"
```

Comment:
Checked that the tag is based directly on the bumping of the version to 3.2.0-M1 - Latest commit was the change from SNAPSHOT to M1.